pkg/kp/service.go

```diff
@@ -7,6 +7,12 @@ import (
 	"github.com/square/p2/pkg/pods"
 )
 
+var (
+	// prints the HTTP response on stderr, while exiting 0 if the status code is 200
+	httpStatusCheck  = `if [[ $(curl http://$(hostname):%v/_status -s -o /dev/stderr -w "%%{http_code}") == "200" ]] ; then exit 0 ; else exit 2; fi`
+	httpsStatusCheck = `if [[ $(curl https://$(hostname):%v/_status -s -o /dev/stderr -w "%%{http_code}" --cacert ${SECRETS_PATH}/service2service.ca.pem) == "200" ]] ; then exit 0 ; else exit 2; fi`
```

Comment:
> 1. writing down the path to secrets
We're doing that anyway, but in code instead of in configuration.

> 2. writing down the ID of a pod...
Yes, but it's not a pod ID we'll be imposing on anyone else. We just need to be aware of it in our own bootstrapping process.

> 3. and the pod id you're referencing isn't even your own
Weird yes, but I don't think this is significantly worse than 1. or 2.

> It's still gross, it just moves the grossness into a pod manifest.
At least we can document it so that people writing manifests for their preparer have a path to addressing it. As we've done, we've required people to at the very least 1) write a hook that exposes `$SECRETS_PATH` 2) symlink or otherwise recreate their CA to our PEM name `service2service.ca.pem`. 

> You won't actually notice the missing cert at all unless you attempt to schedule a pod manifest that contains an https health check.
Which we just made the default. As soon as something is scheduled, the healthcheck is going to start failing.